packages/multi-test/src/custom_handler.rs

```diff
@@ -58,3 +58,72 @@ impl<ExecC, QueryC, T: SimpleCustomHandler<ExecC, QueryC>> CustomHandler<ExecC,
         self.query(block, msg)
     }
 }
+
+/// Custom handler implementation panicking on each call. Assuming, that unless specific behavior
+/// is implemented, custom messages should not be send.
+pub struct PanickingCustomHandler;
+
+impl<ExecC, QueryC> SimpleCustomHandler<ExecC, QueryC> for PanickingCustomHandler
```

Comment:
Why even have the `SimpleCustomHandler` trait? Why not just implement `CustomHandler` directly?

Also, I think `CustomHandler` is the only type in this file that should be `pub` (rather than `pub(crate)`). These other types are only meaningful for us to test multi-test itself (not for blockchains that actually use custom types)

Reply:
Yea, probably in contract we should not use in contracts tests. `SimpleCustomHandler` is here to make implementations kind of simpler as I assume that `app` is somehow accessible on the side, but it may be an overkill.